test/test-udp-send-and-recv.c

```diff
@@ -190,6 +198,19 @@ TEST_IMPL(udp_send_and_recv) {
                   cl_send_cb);
   ASSERT(r == 0);
 
+  /* client sends "ping", expects "pong" */
+  buf = uv_buf_init("PING", 4);
+
+  r = uv_udp_send_ex(&req_ex,
+                     &client,
+                     &buf,
+                     1,
+                     (const struct sockaddr*) &addr,
+                     sizeof(struct sockaddr_in),
+                     cl_send_cb);
+  ASSERT(r == 0);
```

Comment:
```suggestion
  ASSERT_EQ(r, 0);
```
Ditto lines 222-225.